File: app/src/main/java/com/shveed/cookmegood/data/UserApi.java
package com.shveed.cookmegood.data;

import com.shveed.cookmegood.entity.User;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface UserApi {

    String API_PATH = "http://10.0.2.2:8080";

    @GET(API_PATH + "/users/Auth")
    Call<User> checkUser(@Query("login") String login,
                         @Query("pass") String pass);


}

File: app/src/main/java/com/cookMeGood/makeItTasteIt/data/UserApi.kt
package com.cookMeGood.makeItTasteIt.data

import com.cookMeGood.makeItTasteIt.data.dto.User
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface UserApi {
    @GET("$API_PATH/users/Auth")
    fun checkUser(@Query("login") login: String?,
                  @Query("pass") pass: String?): Call<User?>?

    companion object {
        const val API_PATH = "http://10.0.2.2:8080"
    }
}